lib/song.rb

```diff
@@ -0,0 +1,12 @@
+# require_relative "../config/environment.rb"
```

Comment:
remove the commented out requires. Don't want to encourage student's bad behavior